x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/inference/action/GetInferenceModelAction.java

```diff
@@ -34,19 +34,40 @@ public GetInferenceModelAction() {
 
     public static class Request extends AcknowledgedRequest<GetInferenceModelAction.Request> {
 
+        private static boolean DEFAULT_DO_NOT_PERSIST_DEFAULT_CONFIGS = false;
+
         private final String inferenceEntityId;
         private final TaskType taskType;
+        // Default endpoint configurations are persisted on first read.
+        // Set to true to avoid persisting on read.
+        // This setting only applies to GET * requests it has
+        // no effect when getting a single model
+        private final boolean doNotPersistDefaultConfigs;
```

Comment:
In general, I dislike negative boolean variable names. Why not name it `persistDefaultConfig`?

Reply:
I flip flopped on this having switched the meaning once already. I'm not sure of the reason why I decided to use the verbose form other that it was friday afternoon. I've renamed to `persistDefaultConfig`